fluentd/configs.d/openshift/input-pre-k8s-audit-log.conf

```diff
@@ -0,0 +1,16 @@
+<source>
+  @type tail
+  @id k8s-audit-input
+  @label @INGRESS
+  @log_level debug 
+  path "#{ENV['K8S_AUDIT_FILE'] || '/var/log/kube-apiserver/audit.log'}"
+  pos_file "#{ENV['K8S_AUDIT_POS_FILE'] || '/var/log/kube-apiserver/audit.log.pos'}"
+  tag audit.log
```

Comment:
We're going to need a separate tag for this - I suggest `audit.k8s` - this also means changes elsewhere in the pipeline - we might be able to change many of the existing uses of `audit.log` to denote operations logs, to use `audit.**` instead which means "consider any log tagged with audit.** to be an operations log"